source/OptimaJet.DWKit.StarterApplication/OptimaJet.DWKit.StarterApplication.csproj

```diff
@@ -61,7 +61,7 @@
     <PackageReference Include="SparkPostDotNet" Version="0.3.0" />
     <PackageReference Include="Hangfire.AspNetCore" Version="1.6.20" />
     <PackageReference Include="Hangfire.PostgreSql" Version="1.4.8.2" />
-    <PackageReference Include="JsonApiDotNetCore" Version="3.0.0" />
+    <PackageReference Include="JsonApiDotNetCore" Version="3.1.0" />
```

Comment:
This is causing Email to fail processing with this exception:

```
System.InvalidOperationException: Cannot find compilation library location for package 'JsonApiDotNetCore'
   at Microsoft.Extensions.DependencyModel.CompilationLibrary.ResolveReferencePaths(ICompilationAssemblyResolver resolver, List`1 assemblies)
   at System.Linq.Enumerable.SelectManySingleSelectorIterator`2.MoveNext()
   at RazorLight.Compilation.DefaultMetadataReferenceManager.Resolve(DependencyContext dependencyContext)
   at RazorLight.Compilation.RoslynCompilationService.EnsureOptions()
   at RazorLight.Compilation.RoslynCompilationService.get_ParseOptions()
   at RazorLight.Compilation.RoslynCompilationService.CreateSyntaxTree(SourceText sourceText)
   at RazorLight.Compilation.RoslynCompilationService.CreateCompilation(String compilationContent, String assemblyName)
   at RazorLight.Compilation.RoslynCompilationService.CompileAndEmit(IGeneratedRazorTemplate razorTemplate)
   at RazorLight.Compilation.RoslynCompilationService.CompileAsync(IGeneratedRazorTemplate razorTemplate)
--- End of stack trace from previous location where exception was thrown ---
   at RazorLight.Compilation.TemplateFactoryProvider.CompileAsync(IGeneratedRazorTemplate razorTemplate)
   at RazorLight.Compilation.TemplateFactoryProvider.CreateFactoryAsync(String templateKey)
   at RazorLight.RazorLightEngine.CompileTemplateAsync(String key)
   at RazorLight.RazorLightEngine.CompileRenderAsync(String key, Object model, Type modelType, ExpandoObject viewBag)
   at FluentEmail.Razor.RazorRenderer.ParseAsync[T](String template, T model, Boolean isHtml)
   at FluentEmail.Razor.RazorRenderer.FluentEmail.Core.Interfaces.ITemplateRenderer.Parse[T](String template, T model, Boolean isHtml)
   at FluentEmail.Core.Email.UsingTemplateFromFile[T](String filename, T model, Boolean isHtml)``` 

I am looking into this.

Reply:
this could be a caching issue (most common problem with Docker)

Reply:
I can't reproduce.  I assume it was some issue with my license.key file being missing and having a weird affect.